mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -184,7 +241,23 @@ def transformation():  # pylint: disable=unused-variable
             data = flask.request.data.decode("utf-8")
             csv_input = StringIO(data)
             data = parse_csv_input(csv_input=csv_input)
-        elif flask.request.content_type in [CONTENT_TYPE_JSON, CONTENT_TYPE_JSON_SPLIT_ORIENTED]:
+        elif flask.request.content_type == CONTENT_TYPE_JSON:
+            json_str = flask.request.data.decode("utf-8")
+            try:
+                inp_dict = json.loads(json_str)
```

Comment:
the input does not have to be a dict, it can also be a list in case the data is in orient='records'